doc/content/getting-started/packet-broker/configure/routing-policies.md

```diff
@@ -20,7 +20,7 @@ This guide explains how to manage routing policies, i.e. the rules for forwardin
 
 1. Understand how Packet Broker works and what routing policies are. [Learn about Packet Broker]({{< relref "../" >}}).
 2. {{% tts %}} connected to Packet Broker.
-  - {{< distributions "Cloud" >}} **{{% tts %}} Cloud** is already connected to Packet Broker.
+  - {{< distributions "Cloud" >}} **{{% tts %}} Cloud** is already connected to Packet Broker. To configure routing to/from another specific Cloud customer, [contact The Things Industries support](mailto:support@thethingsindustries.com).
```

Comment:
Actually this is self service already, but we should make it easier to list networks.

Reply:
I wasn't able to add a specific network using the console

Reply:
It's because those networks aren't listed yet, and it's currently not possible to configure that yourself. That's tracked in https://github.com/TheThingsNetwork/lorawan-stack/issues/4447. Please revert the change on this line; they do not contact us to configure routing

Reply:
Okay, and the support ticket is also incorrect and can just be closed then? -> https://github.com/TheThingsIndustries/lorawan-stack-support/issues/526